ccd-definition/AuthorisationCaseEvent.json

```diff
@@ -345,8 +338,43 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseEventID": "createNoticeOfProceedings",
```

Comment:
you have removed perms here

Reply:
Which is fine, we do not want to go live with createNoticeOfProceedings yet.

Reply:
C and U are easier for me to add back than the full permission